### DISCUSSION TITLE
github görevleri

### DISCUSSION THREAD

**Discussion by MUTOYASAR** [IBAX-io/go-ibax#1044] [Soru Cevap - Türkçe]
github görevleri nerden ve nasıl yapılıyor bilen var mı

**Comment by Yusufknk** [comment]
Githubta star ve work yapmalısınız. Nasıl yapılacağı ile ilgili İbax Türkiye telegram grubunda sabitlenmiş mesajlarda video mevcut.

**Reply by Eymenedip** [reply]
Kim biliyor

**Comment by hes91** [comment]
Airdrop sayfasında github bölümünde star ve fork yapmalısınız.

**Comment by MUTOYASAR** [comment]
star ve fork tan 3000 point veriyor onun dışında githubda nasıl işlemler yaparsak github coins yani İBXC verir bilen var mı

**Reply by Eymenedip** [reply]
Kaç puanın var
